<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckMulticollinearityDialog</class>
 <widget class="QWidget" name="CheckMulticollinearityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1424</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1424</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multicollinearity assessment tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>

     <widget class="QWidget" name="tabMulticoll">
      <attribute name="title">
       <string>Multicollinearity assessment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitterMain">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>8</number>
         </property>
         <widget class="QWidget" name="topPanel">
          <layout class="QVBoxLayout" name="topPanelLayout">
           <item>
            <layout class="QHBoxLayout" name="maskLayerLayout">
             <item>
              <widget class="QLabel" name="labelMask">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Optional study area mask:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cboMaskLayer">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkEnableVIF">
               <property name="text">
                <string>Compute Variance Inflation Factor (VIF) (optional)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lblInstructions">
             <property name="styleSheet">
              <string notr="true">background-color: transparent; border: none; color: #444; font-style: italic;</string>
             </property>
             <property name="text">
              <string>Please select the layers you want to assess for multicollinearity. For the test to work, all layers must be aligned.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layerLayout">
             <item>
              <widget class="QTreeWidget" name="treeAvailableLayers">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="headerHidden">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="controlLayout">
               <item>
                <widget class="QPushButton" name="btnAdd">
                 <property name="text">
                  <string>Add →</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRemove">
                 <property name="text">
                  <string>← Remove</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="selectedLayout">
               <item>
                <widget class="QTreeWidget" name="treeSelectedLayers">
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                 </property>
                 <property name="headerHidden">
                  <bool>true</bool>
                 </property>
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnVerifyAlignment">
                 <property name="text">
                  <string>Verify the alignment of the selected layers</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="buttonLayout">
             <item>
              <widget class="QPushButton" name="btnRun">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Verify multicollinearity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCancel">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="progressBarLayout">
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="visible">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QTabWidget" name="tabResults">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabSummary">
           <attribute name="title">
            <string>Summary (Spearman correlation + VIF)</string>
           </attribute>
           <layout class="QVBoxLayout" name="summaryLayout">
            <item>
             <widget class="QTextBrowser" name="htmlSummary"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabMatrix">
           <attribute name="title">
            <string>Spearman Correlation Matrix</string>
           </attribute>
           <layout class="QVBoxLayout" name="matrixLayout">
            <item>
             <widget class="QTextBrowser" name="htmlMatrix"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabVIF">
           <attribute name="title">
            <string>Variance Inflation Factor (VIF)</string>
           </attribute>
           <layout class="QVBoxLayout" name="vifLayout">
            <item>
             <widget class="QTextBrowser" name="htmlVIF"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="exportLayout">
         <item>
          <widget class="QPushButton" name="btnExportCSV">
           <property name="text">
            <string>Export CSV (corr.)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExportHTML">
           <property name="text">
            <string>Export HTML (corr)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCopyClipboard">
           <property name="text">
            <string>Copy to Clipboard (corr)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExportVifCSV">
           <property name="text">
            <string>Export CSV (VIF)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExportVifHTML">
           <property name="text">
            <string>Export HTML (VIF)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>

    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
